src/constants/constants.ts

```diff
@@ -179,7 +179,8 @@ export const LSP6_DEFAULT_PERMISSIONS = {
   DECRYPT                          : "0x0000000000000000000000000000000000000000000000000000000000100000",
   SIGN                             : "0x0000000000000000000000000000000000000000000000000000000000200000",
   EXECUTE_RELAY_CALL               : "0x0000000000000000000000000000000000000000000000000000000000400000",
-  ERC4337_PERMISSION               : "0x0000000000000000000000000000000000000000000000000000000000800000"
+  ERC4337_PERMISSION               : "0x0000000000000000000000000000000000000000000000000000000000800000",
```

Comment:
Suggest adding a comment that this is not a standard permission defined in LSP6, but used for integration of ERC4337 standard.

Otherwise, people might look into the LSP6 specs to understand what this is used for, but will not find it.

Do we have a link in our docs or a guide that mentions this permission? Maybe worth adding it here.

@Hugoo @skimaharvey wdyt?

Reply:
i agree that this is confusing. I am not sure if the confusion starts at the lib level but imo more at the LIPs level and probably we should fix it there first